README.md

```diff
@@ -159,12 +161,22 @@ Note you must provide exactly one of the `token`, `serial`, or `slot-id` attribu
 
 Private keys can also be stored in Azure's [key management offerings](https://docs.microsoft.com/en-us/azure/key-vault/keys/about-keys).
 ```
+private_key_stores:
     - uri: https://keyless-hsm-1.managedhsm.azure.net/keys/keyless-a/256400ae07e74327b5d233c15aea837
     - uri: https://keyless-vault-1.vault.azure.net/keys/keyless-b/d791e7f42b3a4f3ea8acc65014ea6a95
 ```
 If gokeyless is running in a VM with Managed Services enabled, auth works out of the box. Otherwise, credentials can also be specified with an env var containing the path to a file. (env vars are defined [here](https://pkg.go.dev/github.com/Azure/go-autorest/autorest/azure/auth#pkg-constants))
 The required roles are `/keys/read/action` and `/keys/sign/action`
 
+### Google Cloud KMS
+
+Private keys can also be stored in Google Cloud's [key management offerings](https://cloud.google.com/security-key-management)
+```
+private_key_stores:
+    - uri: projects/abc/locations/us-west1/keyRings/xyz/cryptoKeys/example-key/cryptoKeyVersions/3
```

Comment:
so they actually do have a full URI format, but it's _schemaless_ https://cloud.google.com/apis/design/resource_names#full_resource_name. That being said, it's not supported for the KMS API (`name does not match pattern: /^projects\/[^/]+\/locations\/[^/]+\/keyRings\/[^/]+\/cryptoKeys\/[^/]+\/cryptoKeyVersions\/[^/]+$/`), only for these: https://cloud.google.com/iam/docs/full-resource-names and for IAM usage. That being said, I think this is fine as-is since we can strengthen the `IsKMS` resource check if we ever add any additional providers that get even close to conflicting, which is probably unlikely since everyone names everything slightly differently.

Reply:
I feel sad now